setup.py

```diff
@@ -11,15 +11,15 @@
 with open("HISTORY.rst") as history_file:
     history = history_file.read()
 
-requirements = ["pandas>=0.25.0", "requests", "typing_extensions"]
+requirements = ["pandas>=1.0.0", "requests", "typing_extensions"]
```

Comment:
Is this strictly necessary? I only mean that pandas 1.0.0 is not around for too long.

Reply:
By now we had some changes that make it necessary. The `json_normalize` function moved around in the pandas sub modules and the new import location is used. (The old one would still work but yields a depreciation warning). Strictly speaking this is the only function that enforces fairly new pandas requirements on us. It was also the reason for the 0.25 requirement. Otherwise we could have dropped all the way down to 0.22 I think.
Nonetheless I don't think it is much of an issue as pandas >= 1 still supports python starting from 3.6.1 which is essentially what we want to support anyways, so a fresh environment should work for most people. If we get issues or requests for lower pandas versions we could work on it. But until then I would not prioritize it very much.